cloudinit/util.py

```diff
@@ -1279,6 +1284,26 @@ def find_devs_with_netbsd(criteria=None, oformat='device',
     return []
 
 
+def find_devs_with_openbsd(criteria=None, oformat='device',
+                           tag=None, no_cache=False, path=None):
+    out, _err = subp(['sysctl', '-n', 'hw.disknames'], rcs=[0])
```

Comment:
This time, the function comes with a unit-test, and is in cloudinit.util. This way, I don't need to patch `cloudinit/sources/DataSourceNoCloud.py`.

Reply:
Thanks for this!